src/inspector/view/qml/MuseScore/Inspector/popups/text/TextStylePopup.qml

```diff
@@ -269,9 +282,14 @@ StyledPopupView {
                         transparent: true
 
                         iconCode: modelData["iconRole"]
-                        checked: false //currentSubscriptOption(/*TODO*)
+                        checked: textStyleModel.textSettingsModel && !textStyleModel.textSettingsModel.textScriptAlignment.isUndefined ? textStyleModel.textSettingsModel.textScriptAlignment.value === modelData["typeRole"]
+                                                                                           : false
```

Comment:
Should I remove null checks from everywhere?

Reply:
If it indeed works without them, then yes :)
(and I'm quite sure it will work, because `textStyleModel` is just the ID of a component, so that's never null, and `textStyleModel.textSettingsModel` won't be null either because the textSettingsModel property is initialised directly in the constructor of TextStylePopupModel.)